includes/stats-calculator.php

```diff
@@ -187,6 +188,55 @@ function ( $a, $b ) {
 		return $users;
 	}
 
+	/**
+	 * Get projects for an event.
+	 */
+	public function get_projects( WP_Post $event ): array {
```

Comment:
Instead of passing the whole object here, we could pass just the id as the function only needs the id.

Reply:
Thanks